tests/test_model_loading.py

```diff
@@ -26,11 +27,14 @@ def test_model_loading_via_LocalFileHandler(path: str | pathlib.Path):
     capellambse.MelodyModel(path)
 
 
-def test_model_loading_via_GitFileHandler():
-    path = "git+" + pathlib.Path.cwd().as_uri()
-    capellambse.MelodyModel(
-        path, entrypoint="tests/data/melodymodel/5_0/Melody Model Test.aird"
-    )
+class TestGitFileHandler:
```

Comment:
> On one of our production repositories, this reduces the amount of data
transferred by about 90%, and the time until the `MelodyModel` object
becomes ready during an initial clone by roughly the same amount.

I'm convinced that the network-bandwidth reduction happens b/c I trust in git. But I can remember that I checked your claim regarding space benefits and couldn't prove it. Do you want to prove it with either a showcase or a dedicated test case right here? I promise this is the last time I raise this.

Reply:
I sense dark magic here. You can pass.